docs/generators/haskell.md

```diff
@@ -195,7 +195,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |BasicAuth|✓|OAS2,OAS3
 |ApiKey|✓|OAS2,OAS3
 |OpenIDConnect|✗|OAS3
-|BearerToken|✗|OAS3
+|BearerToken|✓|OAS3
```

Comment:
what is this change for?

Reply:
Doh... thought I had reverted that. It's the result of one of the scripts they want ran before a PR.  I'll revert it and then merge.